package.json

```diff
@@ -3,11 +3,8 @@
   "version": "1.0.0",
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "test": "gulp ci"
+    "test": "lab test/* --coverage && standard"
```

Comment:
I like the opinionated-ness of `standard`. And aside from that the style is good.  

Particularly for an open source lib, it's one less moving part.